bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -430,8 +430,13 @@ impl Client {
         })
     }
 
-    pub fn account_url(&self) -> Option<String> {
-        self.inner.oidc().account_management_url().ok().flatten().map(|url| url.to_string())
+    pub fn account_url(&self, action: Option<AccountManagementAction>) -> Option<String> {
+        self.inner
+            .oidc()
+            .account_management_url(action.map(Into::into))
+            .ok()
```

Comment:
Instead of this `ok()` call, can you use a match, and log the error in the `Err` case please?